staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/generic/interfaces.go

```diff
@@ -49,6 +49,9 @@ type Source[H Hook] interface {
 // Dispatcher dispatches evaluates an admission request against the currently
 // active hooks returned by the source.
 type Dispatcher[H Hook] interface {
+	// Run the dispatcher. This method should be called only once at startup.
+	Run(ctx context.Context) error
```

Comment:
it blocks? In most case in the codebase Run=blocks, Start=returns

Reply:
Looks like it starts goroutines and routines.

@alexzielenski does this match your understanding?  Should we rename?

Reply:
Great, thanks for confirming. I'll make the edit